client/src/store/index.ts

```diff
@@ -66,7 +67,7 @@ function getArrayName(filename) {
 }
 
 function getData(id, file, webWorker = null) {
-  return new Promise((resolve, reject) => {
+  return new BluebirdPromise((resolve, reject) => {
```

Comment:
Do you know why we were even using bluebird in the first place?

Reply:
No idea, we could maybe ask Scott. Rather than figure it out/exorcise it I just did the minimum to stop the TS errors.

Reply:
Ok, I'll file a ticket. It doesn't look like it's required for either itk.js or vtk.js, so hopefully we can rip it out.